helm-chart/kuberay-operator/values.yaml

```diff
@@ -36,10 +36,12 @@ resources:
   # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
   limits:
     cpu: 100m
```

Comment:
Should we increase this too?
Maybe 200m?

Reply:
The concern here is that the CPU-constrained CI deploys the operator with these default resource limits.
I think it's fine to leave as is for now, to avoid worrying about that.